docs/entity_media_player.md

```diff
@@ -11,6 +11,20 @@ sidebar_label: Media Player
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
+| sound_mode | string | None | The current sound mode of the media player
+| sound_mode_list | list | None | Dynamic list of available sound modes (set by platform)
+| support_select_sound_mode | boolean | False | Boolean if select sound mode command supported
```

Comment:
Should this be a flag in `supported_features` or just decided upon whether the `sound_mode_list` is empty or not? The example below should also show the way how to do it (and also include the sound_mode & sound_mode_list methods, right?

Reply:
Yeah, this should not be it's own boolean.

Reply:
I removed it.
I agree it makes more sense to just look if sound_mode_list is empty or not.